### PR TITLE
Add provinces to Australian labels

### LINKED ISSUE #615: Add region to AUS labels
Regions should be added to Australian labels for disambiguation.

### PR DESCRIPTION
**Pull request by avulfson17**
Fixes #615

### CODE REVIEW

File: helper/labelSchema.js [hunk 1]
```diff
@@ -13,12 +13,17 @@ module.exports = {
   'USA': {
     'borough': getFirstProperty(['borough']),
     'local': getFirstProperty(['locality', 'localadmin', 'county']),
-    'regional': getUsOrCaState,
+    'regional': getUsOrCaOrAusState,
```
Comment (orangejulius) [inline helper/labelSchema.js]: after how many countries do we rename this to something like `getRegionalValue`? 3 feels like a good time :)

File: helper/labelSchema.js [hunk 2]
```diff
@@ -13,12 +13,17 @@ module.exports = {
...
     'country': getUSACountryValue
   },
+  'AUS': {
+    'local' : getFirstProperty(['locality', 'localadmin']),
+    'regional' : getUsOrCaOrAusState,
```
Comment (avulfson17) [inline helper/labelSchema.js]: is there a reason that this function isn't called here? why does it just store the variable name?

**Comment by orangejulius** [comment]
LGTM now!

**Comment by trescube** [comment]
:shipit:

File: helper/labelSchema.js (hunk 2 above)
Comment (orangejulius) [inline helper/labelSchema.js]: There is a reason! It's a bit confusing. The labelGenerator.js module which actually generates the labels loads this module, and [picks the correct country's schema to use](https://github.com/pelias/api/blob/master/helper/labelGenerator.js#L41-L48), and then calls [the function from that schema](https://github.com/pelias/api/blob/master/helper/labelGenerator.js#L12-L13)

**Pull request merged**